api/provider/aws/test_aws/environment_scaler_test.go

```diff
@@ -0,0 +1,10 @@
+package test_aws
+
+import (
+	"testing"
+)
+
+// Add Old Test Verbatim from Dev PR
```

Comment:
Thinking those tests will need to be totally rewritten, I don't know how worthwhile they were to begin with.

Reply:
I’ve already started on testing and I’m using all the mockaws calls. I’ll put something up today to view. However I’m waiting on tlake to finish scaler so it’ll have to be adjusted. Also breaking up the code will help.

Reply:
I totally disagree with @bivers. The old tests only testing the scaling algorithm (these need to be properly split up to achieve that), and had a ton of test cases. It's very difficult to get all of the old test cases to pass, because they were specifically designed to trip up a non-optimal algorithm. They should be added verbatim, or one needs to make a strong case as to why any should be left out.